### ISSUE TITLE
imbalanced classification during evidence extraction of HotpotQA

### ISSUE THREAD

**Issue by Fan-Luo** [allenai/longformer#188]
Hi,

> For evidence extraction we apply 2 layer feedforward networks on top of the representations corresponding to sentence and paragraph tokens to get the corresponding evidence prediction scores and use binary cross entropy loss to train the model.

May I ask did you do binary classification for each sentence or paragraph?
Since there are 2 positives and 8 negatives for paragraph prediction, and similarly ~2 positives and >>2 negatives for sentence prediction, did you use under-sample/oversample or weight the loss?

Thank you

**Comment by armancohan** [comment]
We did not use any under/over sampling or weighting loss approaches.

**Comment by Fan-Luo** [comment]
Thank you for your reply.
